### PR TITLE
add Giant Mammoth Chain

### PR DESCRIPTION
**Pull request by MammothDevMaster**
add Giant Mammoth Chain

### CODE REVIEW

File: _data/icons/gmmt.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+    {
+      "url" : " ipfs://QmVth4aPeskDTFqRifUugJx6gyEHCmx2PFbMWUtsCSQFkF" ,
+      "width": 468,
+      "height": 518,
+      "format": "png"
+    }
+  ]
+
```
Comment (ligi) [inline _data/icons/gmmt.json:9]:
```suggestion
```

File: _data/chains/eip155-8989.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    "infoURL": "https://gmmtchain.io/",
+    "shortName": "gmmt",
+    "chainId": 8989,
+    "networkId": 8989,
+    "icon": "gmmt",
+    "explorers": [
+      {
+        "name": "gmmtscan",
+        "url": "https://scan.gmmtchain.io/",
```
Comment (ligi) [inline _data/chains/eip155-8989.json:22]:
```suggestion
        "url": "https://scan.gmmtchain.io",
```

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/icons/gmmt.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[
+    {
+      "url" : " ipfs://QmVth4aPeskDTFqRifUugJx6gyEHCmx2PFbMWUtsCSQFkF" ,
```
Comment (ligi) [inline _data/icons/gmmt.json:3]:
```suggestion
      "url" : "ipfs://QmVth4aPeskDTFqRifUugJx6gyEHCmx2PFbMWUtsCSQFkF" ,
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Comment by MammothDevMaster** [comment]
Merge and prettier update Finish

File: _data/icons/gmmt.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": " ipfs://QmVth4aPeskDTFqRifUugJx6gyEHCmx2PFbMWUtsCSQFkF",
```
Comment (ligi) [inline _data/icons/gmmt.json:3]:
```suggestion
    "url": "ipfs://QmVth4aPeskDTFqRifUugJx6gyEHCmx2PFbMWUtsCSQFkF",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — APPROVED** [review]

**Pull request merged**